themes/src/main/resources/theme/base/login/webauthn-register.ftl

```diff
@@ -21,158 +21,10 @@
             </div>
         </form>
 
-        <script type="text/javascript" src="${url.resourcesCommonPath}/node_modules/jquery/dist/jquery.min.js"></script>
-        <script type="text/javascript" src="${url.resourcesPath}/js/base64url.js"></script>
+        <script type="module" src="${url.resourcesPath}/js/webauthnRegister.js"></script>
```

Comment:
Rather than loading this as a script, it might be preferential to use the `import` statement in the script below to import the functions you need. For example:
```js
import {registerByWebAuthn} from "${url.resourcesPath}/js/webauthnRegister.js";
```